studio-docs/source/explorer.mdx

```diff
@@ -262,6 +264,14 @@ As you build your query, the Explorer runs partial queries under the hood and sh
 
 The Explorer does not show response hints for mutations (this requires running partial mutations, which is unsafe).
 
+#### Default Headers
+
+> Set **Default Headers** from the Explorer's Settings tab.
+
+> **WARNING:** Default Headers should not be used to store sensitive information. Users should use the **Environment Variables** pane to set these.
```

Comment:
"**Do not use default headers to provide sensitive information**, such as an access token. Instead, use [environment variables](#environment-variables)."